public/lib/uploadHttpConcurrentProgress.js

```diff
@@ -44,7 +44,7 @@
         this.clientCallbackProgress = callbackProgress;
         this.clientCallbackError = callbackError;
 
-        this.movingAverage = 10;
+        this.movingAverage = 2;
```

Comment:
downloadHttpConcurrentProgress passes movingAverage as function argument call. Do we need to have this hardcoded here? I think it's better as a library function to have that passed as argument and change the value in the tests (or pass 2 as default value that can be overwritten)